uvicorn/config.py

```diff
@@ -83,7 +83,6 @@ def __init__(
         timeout_keep_alive=5,
         timeout_notify=30,
         callback_notify=None,
-        install_signal_handlers=True,
```

Comment:
Not sure if we want to remove it from the arguments (wouldn't be fully backwards compatible)

Reply:
It's not documented API, so we're good.
(We were just using it internally)